src/Atc.Hosting/BackgroundServiceBaseLoggerMessages.cs

```diff
@@ -2,55 +2,57 @@
 namespace Atc.Hosting;
 
 /// <summary>
-/// BackgroundServiceBase - Logging.
+/// BackgroundServiceBaseLoggerMessages
 /// </summary>
-[SuppressMessage("Design", "MA0048:File name must match type name", Justification = "OK.")]
-[SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "OK")]
-[SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1619:GenericTypeParametersMustBeDocumentedPartialClass", Justification = "OK.")]
-public partial class BackgroundServiceBase<T>
+internal static partial class BackgroundServiceBaseLoggerMessages
 {
-    [SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "OK.")]
-    [SuppressMessage("Design", "CA1051:Do not declare visible instance fields", Justification = "OK.")]
-    [SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1401:Fields should be private", Justification = "OK.")]
-    protected readonly ILogger logger;
-
     [LoggerMessage(
         EventId = LoggingEventIdConstants.BackgroundServiceStarted,
         Level = LogLevel.Information,
-        Message = "Starting worker {serviceName}. Worker will run with {repeatIntervalSeconds} seconds interval")]
-    private partial void LogBackgroundServiceStarted(
+        Message = "Starting worker {serviceName}. Worker will run with {repeatIntervalSeconds} seconds interval",
+        SkipEnabledCheck = false)]
+    internal static partial void LogBackgroundServiceStarted(
+        this ILogger logger,
```

Comment:
After changeing this class from a partial class into a static class with all methods to extension-methods, then this class should be renamed and moved to a correct folder structure like /Extensions/Internal/LoggerExtensions.cs
But why in the first place change it, since is a wellknow parttern for logging with source generatiors?